cloud/client.go

```diff
@@ -993,7 +993,8 @@ func (c *client) LaunchSatellite(name, orgID string) (*SatelliteInstance, error)
 		Name:     name,
 		Platform: "linux/amd64", // TODO support arm64 as well
 	}
-	status, body, err := c.doCall("POST", "/api/v0/satellites", withAuth(), withJSONBody(&req))
+	status, body, err := c.doCall("POST", "/api/v0/satellites",
+		withAuth(), withHeader("Grpc-Timeout", "5M"), withJSONBody(&req))
```

Comment:
can you move the `"5M"` to a variable under the client struct, e.g. `satelliteMgmtTimeout`, that way we can update this value in a single location if we need to change it.

```suggestion
		withAuth(), withHeader("Grpc-Timeout", c.satelliteMgmtTimeout), withJSONBody(&req))
```

or do you foresee us having to configure these independently?

Reply:
I can do that. In theory, they could be different since the delete is a bit longer than the launch. But I think it's just simpler to leave them the same for now.

Reply:
Preferred setting it as a constant especially since we were already using a constant for a similar type of thing